liquidhaskell-boot/src/Language/Haskell/Liquid/LHNameResolution.hs

```diff
@@ -682,6 +685,16 @@ resolveLogicNames cfg env globalRdrEnv unhandledNames lmap0 localVars lnameEnv p
     findReflection :: GHC.Name -> Maybe LHName
     findReflection n = GHC.lookupNameEnv (lneReflected lnameEnv) n
 
+mapMeasureNamesM :: Monad m => (LocSymbol -> m LocSymbol) -> MeasureV v ty ctor -> m (MeasureV v ty ctor)
```

Comment:
This is smelling a lot like lenses!

Reply:
Yeah. I'm not eager to introduce the lense machinery for a few queries. But I'm open to be convinced that there are sufficient use opportunities to be worth it.